src/OpenLoco/src/Audio/Audio.cpp

```diff
@@ -26,6 +26,7 @@
 #include <array>
 #include <cassert>
 #include <unordered_map>
+#include <numeric> // std::iota
```

Comment:
Remove the comment, if we were to start commenting all use it would get messy real quick.